state/documentdb.yaml

```diff
@@ -0,0 +1,171 @@
+---
+# Copyright 2020 widdix GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'State: DocumentDB cluster, a cloudonaut.io template'
+Metadata: {} # FIXME
+Parameters:
+  ParentVPCStack:
+    Description: 'Stack name of parent VPC stack based on vpc/vpc-*azs.yaml template.'
+    Type: String
+  ParentClientStack:
+    Description: 'Stack name of parent client stack based on state/client-sg.yaml template.'
+    Type: String
+  ParentBastionStack:
+    Description: 'Optional but recommended stack name of parent bastion host/instance stack based on vpc/vpc-*-bastion.yaml template.'
+    Type: String
+    Default: ''
+  ParentKmsKeyStack:
+    Description: 'Optional Stack name of parent KMS key stack based on security/kms-key.yaml template (ignored when DBSnapshotIdentifier is set, value used from snapshot).'
+    Type: String
+    Default: ''
+  BackupRetentionPeriod:
+    Description: 'The number of days for which automated backups are retained.'
+    Type: Number
+    MinValue: 1
+    MaxValue: 35
+    Default: 30
+  DBClusterParameterGroupName:
```

Comment:
What about adding a `AWS::DocDB::DBClusterParameterGroup` resource as well? Might be interesting to be able to modify the `change_stream_log_retention_duration`, for example. (see https://docs.aws.amazon.com/documentdb/latest/developerguide/cluster_parameter_groups-list_of_parameters.html)

Reply:
done :)